### DISCUSSION TITLE
Smooth scrolling or current implementation?(POLL)

### DISCUSSION THREAD

**Discussion by txtyash** [helix-editor/helix#3746] [Polls]
In helix one could scroll using half pageup and half page down. It does not use smooth scrolling.
[Here's an example of smooth scrolling](https://github.com/karb94/neoscroll.nvim)

*I'm just a helix user. I'm not a contributor. I put this poll just to get to know people's opinions and to bring attention to #1671*

**Comment by sohzm** [comment]
Any progress?

**Comment by thomasschafer** [comment]
You can enable smooth scrolling with this plugin: https://github.com/thomasschafer/smooth-scroll.hx
